packages/block-editor/src/components/block-preview/style.scss

```diff
@@ -11,8 +11,9 @@
 	overflow: hidden;
 
 	.block-editor-block-preview__content {
-		// This element receives inline styles for width, height, and transform-scale.
+		// This element receives inline styles for transform-scale and aspect-ratio.
 		// Those inline styles are calculated to fit a perfect thumbnail.
+		width: 100%;
```

Comment:
I'm not sure if this breaks other usages of this component elsewhere. I've tested it both in the patterns screen and in the inserter, but there might be more.